tests/openqa/webui/test_results.pm

```diff
@@ -46,7 +46,11 @@ sub run {
     handle_notify_popup;
     assert_screen 'openqa-tests';
     assert_and_click 'openqa-tests';
-    assert_and_click 'openqa-job-minimalx';
+    # At this point the openQA job might still be running or already finished.
+    # Ensure to show finished results at the bottom of the screen whenever the
+    # page finished loading
+    send_key_until_needlematch 'openqa-job-minimalx', 'end';
```

Comment:
IIRC, send)key_until_needlematch presses the key before checking the current screen, right? so in essence this carries the risk that minor UI changes makes the text disappear at the top of the screen already

Reply:
yes, that could happen. As https://openqa.opensuse.org/tests/3336705#step/test_results/14 shows we just still see the "running" result. a "minor UI change" could of course fail the test but we can't know all ways that tests would fail on UI changes so I wouldn't bother. An alternative I considered was to try to wait for the browser to finish loading the page but I wouldn't trust that to work consistently either. The other only other alternative I could think of is to change openQA itself to e.g. jump to an anchor, waste less screen estate, collapse empty sections, etc.